integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXEncryptedVolumeTest.java

```diff
@@ -250,9 +250,8 @@ public void testCreateDistroXWithEncryptedVolumesInSingleRG(TestContext testCont
 
     private ResourceGroup createResourceGroupForEnvironment(TestContext testContext) {
         resourceGroupForTest = resourcePropertyProvider().getName();
-        String username = StringUtils.substringBefore(testContext.getActingUserCrn().getResource(), "@").toLowerCase();
         Map<String, String> tags = new HashMap<>() {{
-            put("owner", username);
+            put("owner", testContext.getActingUserName().split("@")[0].toLowerCase().replaceAll("[^\\w]", "-"));
```

Comment:
Is this logic not needed somewhere else? Would it make sense to wrap it at least as a method in testContext?